backend/config/urls.py

```diff
@@ -3,13 +3,15 @@
 from django.urls import include, path
 
 from hitas.views.health import HealthCheckView
+from users.views import UserInfoView
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/v1/", include("hitas.urls")),
     path("healthz", HealthCheckView.as_view(), name="health_check"),
     path("pysocial/", include("social_django.urls", namespace="social")),
     path("helauth/", include("helusers.urls")),
+    path("helauth/userinfo/", UserInfoView.as_view(), name="userinfo"),
```

Comment:
Even though it is related to helauth, as an API endpoint, this should be under `api/v1/` 🙂

Reply:
I disagree. The endpoint should only be used with users authenticated with Helsinki profile, and the endpoint path reflects that. I think of it as an extension to Helsinki profile rather than a Hitas-specific thing. It's also not something that would be versioned with the rest of the endpoints, so I don't see why it would be under the version path from that perspective either.